src/spec/ScaleSpec.ts

```diff
@@ -0,0 +1,58 @@
+/// <reference path="jasmine.d.ts" />
+/// <reference path="support/js-imagediff.d.ts" />
+/// <reference path="require.d.ts" />
+/// <reference path="TestUtils.ts" />
+/// <reference path="Mocks.ts" />
+
+describe('A scaled and rotated actor', () => {
+   var actor: ex.Actor;
+   var engine: ex.Engine;
+   var scene: ex.Scene;
+   var mock = new Mocks.Mocker();
+   
+   beforeEach(() => {
+      jasmine.addMatchers(imagediff.jasmine);
+
+      actor = new ex.UIActor(50, 50, 100, 50);
+      actor.color = ex.Color.Blue;
+      actor.collisionType = ex.CollisionType.Active;
+      engine = TestUtils.engine();
+
+      
+      scene = new ex.Scene(engine);
+      engine.currentScene = scene;
+
+      spyOn(scene, 'draw').and.callThrough();
+      spyOn(actor, 'draw').and.callThrough();     
+		
+   });
+
+   afterEach(() => {
+      engine.stop();
+   });
+	
+   
+   it('is drawn coorectly scaled at 90 degrees', (done) => {            
+
+      let game: ex.Engine = TestUtils.engine({ width: 800, height: 600 });
```

Comment:
The engine is already availabile above, either remove the one that gets created in `beforeEach` or use that one so we don't have two running at once.